### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Alexandr-Pavelev**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -15,5 +15,8 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
 # write your code here
```
Comment (Abnormaltype) [inline app/main.py:18]: remove unnecessary comment

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
remove unnecessary files from PR

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
image
remove unnecessary files from the PR

**Comment by Alexandr-Pavelev** [comment]
I deleted by mistake file "app". How add this file? tell me please

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
It seems that you haven't pushed a solution of this task

**Comment by Alexandr-Pavelev** [comment]
> It seems that you haven't pushed a solution of this task

I push this task in terminal- answer:
"(venv) Anna-Paveleva-5:pythonProject3 mac$ git commit - am 'Solution'
fatal: not a git repository (or any of the parent directories): .git
(venv) Anna-Paveleva-5:pythonProject3 mac$ git push origin develop
fatal: not a git repository (or any of the parent directories): .git"

I don't understand. Tell me what to do step by step pleas

**Comment by Alexandr-Pavelev** [comment]
> > It seems that you haven't pushed a solution of this task
> 
> I push this task in terminal- answer: "(venv) Anna-Paveleva-5:pythonProject3 mac$ git commit - am 'Solution' fatal: not a git repository (or any of the parent directories): .git (venv) Anna-Paveleva-5:pythonProject3 mac$ git push origin develop fatal: not a git repository (or any of the parent directories): .git"
> 
> I don't understand. Tell me what to do step by step pleas

**Comment by Alexandr-Pavelev** [comment]
please provide an answer

**Pull request closed without merging**